docs/development/codeguide.rst

```diff
@@ -135,8 +135,17 @@ Coding Style/Conventions
   <https://www.python.org/dev/peps/pep-0008/>`_. In particular, this includes
   using only 4 spaces for indentation, and never tabs.
 
+  * ``astropy`` itself enforces this style guide using the
+    `black <https://black.readthedocs.io/en/stable/>`_ code formatter.
+
+  * We recognize that sometimes ``black`` will autoformat things in undesirable
+    ways e.g. matricies.  In the cases that ``black`` produces undesirable code
+    formatting, one can wrap code the code in ``# fmt: off`` and ``# fmt: on``
```

Comment:
So they have to be in pairs? What if I only want to disable the one line, is there an equivalent of `# noqa` for `flake8`?

Reply:
Possibly, here is what the docs say: https://black.readthedocs.io/en/stable/faq.html#why-does-my-linter-or-typechecker-complain-after-i-format-my-code

Reply:
I don't think that answers my question. Is there like a `# noblack` or `# fmt: skip` if I only want to opt-out of black for that one line?

Reply:
Oh, I see what you are asking. Yes `# fmt: skip` is the official way to do it per https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#code-style.

Reply:
Great! Perhaps that should be mentioned here too? That might be very common use case in this repo... 💭